packages/driver/cypress/integration/e2e/multi-domain/multi_domain_validation_specs.ts

```diff
@@ -198,6 +198,15 @@ describe('cy.origin', () => {
       .should('equal', 'Welcome TJohnson')
     })
 
+    it('finds the correct spec bridge even if a previous spec bridge host is a subset of the current host', () => {
```

Comment:
Given the PR title, I expect this test or a few tests to explicitly set chromeWebSecurity: false. Given we can't update that mid-test, it seems like a system-test should have been added to validate this??

Reply:
The 'positive' case of the code is tested for each origin. We verify the port, protocol, and that the host ends with the host from the spec bridge. You are correct that the 'negative' is not tested. I made the call that testing this case  did not nessitate the cost of adding a system test (System tests seem expensive to me 🤷‍♂️ ). I can add a system test if the group disagrees with that reasoning.

Reply:
I've updated an existing system test for chromeWebSecurity to also test this case.